crates/host/src/wasmbus/mod.rs

```diff
@@ -3803,64 +3697,66 @@ impl Host {
         trace!(%subject, "handling control interface request");
 
         let res = match (parts.next(), parts.next(), parts.next(), parts.next()) {
-            (Some("auction"), Some("actor"), None, None) => {
+            (Some("actor"), Some("auction"), None, None) => {
                 self.handle_auction_actor(message.payload).await.map(Some)
             }
-            (Some("auction"), Some("provider"), None, None) => {
-                self.handle_auction_provider(message.payload).await
-            }
-            (Some("cmd"), Some(host_id), Some("lp"), None) => Arc::clone(&self)
-                .handle_launch_provider(message.payload, host_id)
-                .await
-                .map(Some),
-            (Some("cmd"), Some(host_id), Some("sa"), None) => self
-                .handle_stop_actor(message.payload, host_id)
+            (Some("actor"), Some("scale"), Some(host_id), None) => Arc::clone(&self)
+                .handle_scale_actor(message.payload, host_id)
                 .await
                 .map(Some),
-            (Some("cmd"), Some(host_id), Some("scale"), None) => Arc::clone(&self)
-                .handle_scale_actor(message.payload, host_id)
+            (Some("actor"), Some("update"), Some(host_id), None) => self
+                .handle_update_actor(message.payload, host_id)
                 .await
                 .map(Some),
-            (Some("cmd"), Some(host_id), Some("sp"), None) => self
+            (Some("provider"), Some("scale"), Some(host_id), None) => self
                 .handle_stop_provider(message.payload, host_id)
                 .await
                 .map(Some),
-            (Some("cmd"), Some(host_id), Some("stop"), None) => self
-                .handle_stop_host(message.payload, host_id)
+            (Some("provider"), Some("auction"), None, None) => {
+                self.handle_auction_provider(message.payload).await
+            }
+            (Some("provider"), Some("launch"), Some(host_id), None) => Arc::clone(&self)
```

Comment:
I know I proposed it in the PR, but is launch weird here? We don't use it anywhere else, and folks type `wash start provider <>` and not launch 🤔 

Internally it looks like we have launch, so providers you launch and actors you scale. I think it might be better to call this just `start`